drivers/net/Kconfig

```diff
@@ -60,6 +60,11 @@ config NETDEV_WIRELESS_HANDLER
 	---help---
 		Enable the wireless handler support in upper-half driver.
 
+config NETDEV_RSS
+	bool "Using hardware RSS (Receive Side Scaling) with network card"
+	default n
+	depends on SMP && NETDEV_IOCTL && NETDEV_WORK_THREAD
```

Comment:
Please include --help-- with more information about what is RSS and when to use

Reply:
done